### DISCUSSION TITLE
deleting review items should also delete corresponding events

### DISCUSSION THREAD

**Discussion by ivanjx** [blakeblackshear/frigate#13554] [Ideas]
when deleting the review items (alerts and detections), the corresponding events (individual detections) should also be deleted so it wont show up on home assistant integration.

**Comment by NickM-27** [comment]
this is not correct. events are 1:1 with objects that are detected. This means that a single object that exists for a long time (for example, a car in the driveway) will be a part of multiple review items. For example you might have a review item of a car arriving and another one of the same car leaving later. Deleting the first review item would then delete the event even though that event still exists in another review item, which would remove important information

**Reply by ivanjx** [reply]
so there is just no way to remove a recording for good? all the reviews that have been deleted are still very much visible in HA integration

**Reply by NickM-27** [reply]
In 0.15 deleting the review item will delete the recordings after the continuous recording retention period is over.
